couchbase/README.md.jinja

```diff
@@ -30,15 +30,7 @@ using the REST API</a>.
 
 ### Creating a Couchbase user for collectd
 
-If you're monitoring a secured Couchbase deployment, it is a good practice to create a Couchbase user with minimal read-only roles, as follows:
-
-```
-db.createUser( {
-  user: "collectd",
-  pwd: "collectd",
-  roles: [ { role: "readAnyDatabase", db: "admin" }, { role: "clusterMonitor", db: "admin" } ]
-});
-```
+Couchbase provides Role-Based Access Control (RBAC) to restrict system access. A Couchbase role permits one or more resources to be accessed according to defined privileges. For more information, see [Roles.](https://docs.couchbase.com/server/6.6/learn/security/roles.html)
```

Comment:
It would be helpful to mention the `data_monitoring` role as having the minimum permissions to use the integration.